### PR TITLE
Test AboutDrawerItem

### PR DESCRIPTION
**Pull request by abarth**
Also, fix a few minor bugs found by the test.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/test/material/about_test.dart [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('AboutDrawerItem control test', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new MaterialApp(
+        title: 'Example app',
```
Comment (Hixie) [inline packages/flutter/test/material/about_test.dart]: Surely this app should be called the Pirate app.

File: packages/flutter/test/material/about_test.dart [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  testWidgets('AboutDrawerItem control test', (WidgetTester tester) async {
+    Future<Null> licenseFuture;
+    LicenseRegistry.addLicense(() {
+      licenseFuture = tester.pumpWidget(new Container());
+      return new Stream<LicenseEntry>.fromIterable(<LicenseEntry>[]);
+    });
+
+    LicenseRegistry.addLicense(() {
+      tester.pumpWidget(new Container());
```
Comment (Hixie) [inline packages/flutter/test/material/about_test.dart]: are you expecting this closure to be executed at all?

File: packages/flutter_test/lib/src/binding.dart [hunk 3]
```diff
@@ -84,7 +84,6 @@ const Size _kDefaultTestViewportSize = const Size(800.0, 600.0);
 abstract class TestWidgetsFlutterBinding extends BindingBase
```
Comment (Hixie) [inline packages/flutter_test/lib/src/binding.dart]: maybe add to the last paragraph here, saying that the "also, the ServicesBinding is not included" or some such?

**Comment by Hixie** [comment]
LGTM

**Pull request merged**